articles/aks/use-managed-identity.md

```diff
@@ -56,6 +56,8 @@ A successful cluster creation using managed identities contains this service pri
 
 > [!NOTE]
 > For creating and using your own VNet, static IP address, or attached Azure disk where the resources are outside of the MC_* resource group, use the PrincipalID of the cluster System Assigned Managed Identity to perform a role assignment. For more information on role assignment, see [Delegate access to other Azure resources](kubernetes-service-principal.md#delegate-access-to-other-azure-resources).
+>
+> Permission grants to cluster Managed Service Identity used by Azure Cloud provider may take up 60 minutes to populate.
```

Comment:
for consistency with the AAD branding we should use Manages Identity. They dropped 'service' long back..

Reply:
Thanks, Saurya. I just updated it.